Jenkinsfile-jdk11

```diff
@@ -72,14 +72,15 @@ pipeline {
         stage('Build') {
             steps {
                 echo 'Building'
-                sh 'mvn -U -B -e clean install -DskipTests apache-rat:check'
+                sh 'mvn -U -B -e apache-rat:check'
+                sh 'mvn -U -B -e ${MVN_LOCAL_REPO_OPT} clean install -DskipTests'
```

Comment:
Is install really needed?

Reply:
just verify would be fine, but I'm not sure why the build and test were broken into different stages originally?

Maybe the stages should be:
* apache-rat:check
* dependency:resolve dependency:plugin-resolve
* verify -Prun-its